smaug/operators/padding_op.h

```diff
@@ -0,0 +1,115 @@
+#ifndef _OPERATORS_PADDING_OP_H_
+#define _OPERATORS_PADDING_OP_H_
+
+#include "smaug/core/backend.h"
+#include "smaug/core/operator.h"
+#include "smaug/core/tensor.h"
+// #include "smaug/core/tensor_utils.h"
+#include "smaug/core/workspace.h"
+
+namespace smaug {
+
+/** \ingroup Operators
+ * \brief Pad a given tensor in different dimension.
```

Comment:
I think this is meant to pad on H and W dimensions. Let's make this more explicit.

Reply:
Also, mention that the padding on either side of all dimensions is assumed to be equal (e.g. 1 element on both left and right sides).

Reply:
Okay!